File: crossecore-java/src/java_/AnonymousClassDeclaration.java
package java_;
public interface AnonymousClassDeclaration 
 extends ASTNode
{
	
	public static com.crossecore.ocl.QuickSet<AnonymousClassDeclaration> allInstances_ = new com.crossecore.ocl.QuickSet<AnonymousClassDeclaration>(AnonymousClassDeclaration.class);
	
		
	public static com.crossecore.ocl.QuickSet<AnonymousClassDeclaration> allInstances(){
		
		com.crossecore.ocl.QuickSet<AnonymousClassDeclaration> result = new com.crossecore.ocl.QuickSet<AnonymousClassDeclaration>(AnonymousClassDeclaration.class);
		result.addAll(AnonymousClassDeclaration.allInstances_);
		
		
		return result;
	}
	
	//public static com.crossecore.ocl.QuickSet<AnonymousClassDeclaration> allInstances = new com.crossecore.ocl.QuickSet<AnonymousClassDeclaration>(AnonymousClassDeclaration.class);
	
	com.crossecore.ocl.OrderedSet<BodyDeclaration> getBodyDeclarations();
	
	ClassInstanceCreation getClassInstanceCreation();
	void setClassInstanceCreation(ClassInstanceCreation value);
	
}

File: crossecore-typescript/java_/AnonymousClassDeclaration.ts
import {OrderedSet} from "ecore/OrderedSet";
import {Set} from "ecore/Set";
import {ClassInstanceCreation} from "./ClassInstanceCreation";
import {ASTNode} from "./ASTNode";
import {BodyDeclaration} from "./BodyDeclaration";

export interface AnonymousClassDeclaration
extends ASTNode

{
	
	bodyDeclarations: OrderedSet<BodyDeclaration>;
	classInstanceCreation:ClassInstanceCreation;
	

}

